modules/highgui/src/window_w32.cpp

```diff
@@ -596,6 +596,94 @@ void cvSetPropTopmost_W32(const char* name, const bool topmost)
     }
 }
 
+double cvGetPropVsync_W32(const char* name)
+{
+#ifndef HAVE_OPENGL
+    CV_UNUSED(name);
+    CV_Error(Error::OpenGlNotSupported, "Library was built without OpenGL support");
+    return -1;
+#else
+    CV_Assert(name);
+
+    if (!name)
+        CV_Error(Error::StsNullPtr, "NULL name string");
+
+    CvWindow* window = icvFindWindowByName(name);
+    if (!window)
+        CV_Error(Error::StsNullPtr, "NULL window");
```

Comment:
You can use CV_Assert too.

Reply:
I've now given that a more helpful "window doesn't exist" message